### DISCUSSION TITLE
When will 3.0.6 released?

### DISCUSSION THREAD

**Discussion by MeisterQ** [espressif/arduino-esp32#10439] [General]
Just wanted to ask when 3.0.6 will be released and IF zigbee will be implemented into this version for ESP32-H2 ?

**Comment by me-no-dev** [comment]
Will be released soon and yes, ZigBee will be part of it.

**Reply by MeisterQ** [reply]
Thank you very much :)

**Comment by Jason2866** [comment]
3.0.7 is out ;-)
